src/app/components/RadioSchedule/ScheduleItemHeader/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { useContext } from 'react';
```

Comment:
Change: 
- Use ServiceContext 
- Translations no longer passed down to component

![image](https://user-images.githubusercontent.com/59573468/115914077-5b95d700-a469-11eb-9c67-9b5851a988af.png)